llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/lower-args-vararg.ll

```diff
@@ -10,55 +10,59 @@ define void @va1arg(ptr %a, ...) {
   ; RV32-NEXT:   liveins: $x10, $x11, $x12, $x13, $x14, $x15, $x16, $x17
   ; RV32-NEXT: {{  $}}
   ; RV32-NEXT:   [[COPY:%[0-9]+]]:_(p0) = COPY $x10
+  ; RV32-NEXT:   [[FRAME_INDEX:%[0-9]+]]:_(p0) = G_FRAME_INDEX %fixed-stack.1
```

Comment:
I assume it would get optimized by this generic combiner.

```
def ptr_add_immed_matchdata : GIDefMatchData<"PtrAddChain">;                     
def ptr_add_immed_chain : GICombineRule<                                         
  (defs root:$d, ptr_add_immed_matchdata:$matchinfo),                            
  (match (wip_match_opcode G_PTR_ADD):$d,                                        
         [{ return Helper.matchPtrAddImmedChain(*${d}, ${matchinfo}); }]),       
  (apply [{ Helper.applyPtrAddImmedChain(*${d}, ${matchinfo}); }])>
```